### ISSUE TITLE
Translation editor meta UTC and Local dates consistency

### ISSUE THREAD

**Issue by pedro-mendonca** [GlotPress/GlotPress-WP#1429]
Currently the translation meta shows both:
- Date added (UTC)
- Date added (local)

Screenshot:
![imagem](https://user-images.githubusercontent.com/7371591/166707865-26c67611-2173-4df0-9751-b6af75756d17.png)

Questions:
1) Are both needed? Why not keep only local server timestamp?
2) If both are to keep, both can be formated with `%1$s at %2$s` as in core.
  2.1) Local time is being calculated in https://github.com/GlotPress/GlotPress/blob/develop/assets/js/editor.js#L33 and do not respect server time. Can be done server side with `get_date_from_gmt()`

**Comment by psmits1567** [comment]
1. In my opinion only the local timestamp is of interest
2. Formatted like the local setting is also preferable, and I think server side would be the best option

**Comment by pedro-mendonca** [comment]
Could it be just `YYYY-MM-DD at HH:MM (Timezone)`  (formated with `date_i18n()` for both date and time) ?

**Comment by pedro-mendonca** [comment]
To add some more to the discussion, here is a screenshot of the translate.wordpress.org template:

![imagem](https://user-images.githubusercontent.com/7371591/166726355-1897e373-3ca9-4d45-94c3-b584266a188a.png)

**Comment by ocean90** [comment]
> Why not keep only local server timestamp?

Because that's not really relevant for the user for a platform like translate.w.org that's used by users around the globe with different timezones. That's what #972 tried to fix.

But I was never a fan of the current implementation because displaying two dates is just a waste of space and has now real benefit, unless your browser has some incorrect timezone settings. It's also limiting us in adding more dates, like the last modified date as done on translate.w.org.

I think this can be easily solved by allowing each user to set date specific settings in their profile. This includes the timezone and a setting for some common date formats like `Y-M-D` or `D.M.Y`.

**Comment by pedro-mendonca** [comment]
I wrote the server timestamp, but I meant the WordPress install configured timezone.
WP.org timezone is UTC.

A user defined timezone is a good idea, if WordPress had such a setting, GlotPress could inherit it.
In this case, should we go for a new GP user setting with the timezones dropdown?

**Comment by amieiro** [comment]
Why not do this in the browser? We only need to get the UTC time from the database (with PHP) and then add the browser offset and the local data and time formats with JavaScript. Something like this (you can test it in the browser console):
```
var d = new Date('2021-03-22 10:00:25');                                   // This data comes from the server.
d.setTime(d.getTime() + d.getTimezoneOffset() * 60000);                    // Adds the local offset.
console.log(d.toLocaleDateString() + ' at ' + d.toLocaleTimeString());     // 22/03/2021 at 09:00:25: Shows the date and the time with the offset and with the local formats.
```

The `Date` object has some interesting methods. The output is adapted to my localization (dd/mm/YYYY and HH:ii:ss) with my time offset in CET (1 hour):

```
d.toLocaleString();       // '22/03/2021, 09:00:25'
d.toLocaleDateString();   // '22/03/2021'
d.toLocaleTimeString();  // -> '09:00:25'
```

**Comment by pedro-mendonca** [comment]
@amieiro isn't that heavier on tables with many rows than processing it server side with PHP?

**Comment by amieiro** [comment]
I don't think so (I am not sure), since we have to process it on one side or the other. We would have to do a performance test, but a priori it seems to me a better option, because it does not need any configuration by the user.

**Comment by pedro-mendonca** [comment]
Ok let's do both and test the performance.
Let me finish the server side with a user timezone date, and then we can compare.

**Comment by amieiro** [comment]
Perfect. If we can translate the computation  from the server to the browser without a lot of overheading, I think it could be interesting, so we improve the UX.

**Comment by pedro-mendonca** [comment]
@amieiro I've added an optional Timezone user setting -> https://github.com/GlotPress/GlotPress/pull/1430#issuecomment-1118032120
You can now compare with the browser side JS.

**Comment by ocean90** [comment]
> Why not do this in the browser? We only need to get the UTC time from the database (with PHP) and then add the browser offset and the local data and time formats with JavaScript.

Isn't that what we already have, in addition to the UTC time?

**Comment by amieiro** [comment]
> > Why not do this in the browser? We only need to get the UTC time from the database (with PHP) and then add the browser offset and the local data and time formats with JavaScript.
> 
> Isn't that what we already have, in addition to the UTC time?

Yes. We are talking about to remove the UTC time and put only the local time with the local format. I think this could be done with JavaScript, as I [mention before](https://github.com/GlotPress/GlotPress/issues/1429#issuecomment-1117892047).
